commands_to_load/Mod.py

```diff
@@ -82,5 +83,398 @@ def __init__(self, bot):
+            setattr(overwrite[ctx.guild.default_role], 'manage_messages', False)
+            setattr(overwrite[ctx.guild.default_role], 'mention_everyone', True)
+            logger.info('[Mod makechannel()] making a hidden channel')
+
+        # Create channel
+        ch = await ctx.guild.create_text_channel(name, overwrites=overwrite)
+        logger.info('[Mod makechannel()] channel "' + name + '" made by ' + str(ctx.author))
+
+        # Send message to council about channel made 
+        council = discord.utils.get(ctx.guild.channels, name="council")
+        eObj = await em(ctx, description=str(ctx.author) + ' made channel: `' + name + '`', footer='Moderator action')
+        if eObj is not False:
+            await council.send(embed=eObj)
+
+    @commands.command()
+    async def clear(self, ctx, numOfMsgs = 10.0):
+        # Deletes the last X (10 by default) msg's from channel
+        # Limits: Max 100 messages at a time AND no messages older than 2 weeks (14 days)
+
+        logger.info('[Mod clear()] clear function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod clear()] invoking command deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod clear()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        numOfMsgs = int(numOfMsgs)
+
+        # Verify args
+        if numOfMsgs > 100 or numOfMsgs < 1: 
+            # Prevents discord.ClientException 
+            eObj = await em(ctx, description='Number of messages to be between 1 and 100 inclusively', footer='Invalid arguments')
+            if eObj is not False:
+                await ctx.send(embed=eObj)
+
+        channel = ctx.channel
+        # Grab the last X messages from the channel regardless of user
+        logger.info('[Mod clear()] grabbing last {} message from {}'.format(numOfMsgs, channel))
+        messages = await channel.history(limit=numOfMsgs).flatten()
+        logger.info('[Mod clear()] messages to be deleted: {}'.format(messages))
+        
+        try:
+            await channel.delete_messages(messages)
+            logger.info('[Mod clear()] messages from {} deleted'.format(channel))
+        except discord.HTTPException:
+            eObj = await em(ctx, description='Messages cannot be older than 2 weeks', footer='Command Error')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=10.0)
+            return
+
+        eObj = await em(ctx, description='{} messages deleted'.format(numOfMsgs), footer='Message will self destruct in 5 ...')
+        if eObj is not False:
+            await ctx.send(embed=eObj, delete_after=5.0)
+
+    @commands.command()
+    async def purge(self, ctx, *args):
+        # Deletes 10 message in channel from user

```

Comment:
why 10?

Reply:
update this to mention that a different number can be passed in, but 10 is the default